sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -69,6 +69,8 @@ export {
 
 export interface ProxyOptions {
   url?: string;
+  username?: string;
+  password?: string;
```

Comment:
Shouldnt this interface live in core-http such that not every sdk have to define it?